docs/self-managed/platform-deployment/overview.md

```diff
@@ -8,13 +8,21 @@ This chapter contains information for users who want to deploy and run Camunda P
 
 ## Components
 
-Camunda Platform 8 includes the following components as Java applications:
+Camunda Platform 8 includes the following components:
 
 - Zeebe Broker and Gateway
 - Operate (requiring Elasticsearch)
 - Tasklist (requiring Elasticsearch)
 - Optimize (requiring Elasticsearch)
 - Identity (requiring Keycloak)
+- Web Modeler (requiring Identity, Keycloak, and PostgreSQL)
+  :::caution Beta Offering
+  Web Modeler Self-Managed is currently offered as a [beta release](../../../reference/early-access#beta)
+  with limited availability for enterprise customers only. It is not recommended for production use.
+  Special [terms & conditions](https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/) apply.
+  :::
+
+All components except Web Modeler are single Java applications.
```

Comment:
I was not sure about this either, but I thought that it wouldn't hurt to keep it.
Maybe it's still useful because it gives you a hint that you can run most of the components as regular Java applications, too?

Reply:
Probably someone may ask "what is Web Modeler then running on?"

It's relevant since later, the docs say

> You can run the Java applications on a local or virtual machine if it provides a supported Java Virtual Machine (JVM).